src/backgrounds/_base.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file Backgrounds
+ * @module Backgrounds
+ * @overview Background and matching text styles for page modules
+ * @todo: remove _backdrops.scss since this replaces it?
+ * @todo: rename TrialWarranty to be generic, update to use Starry Background
+ */
+
+.background--gray {
```

Comment:
I'll use `light` for this, and discuss with becki, as I'm just porting what was in backdrops, and not clear on whether there are other styles I'm excluding.